daemon/algod/api/algod.oas2.json

```diff
@@ -4887,90 +4109,71 @@
       "in": "path",
       "required": true
     },
-    "currency-greater-than": {
```

Comment:
why did these go? Unused?

Reply:
Correct. It looks like we had plans to support a bunch of conditions that we never did.

Reply:
Just to explain their (likely) origin: We have many of the same types in algod.oas2.json and indexer.oas2.json. Maybe they used to be in one file? So the algod type has extra stuff that was only intended for indexer.

When we generate the model types for the Go SDK, the indexer types overwrite the algod types, so there's no need to maintain the extra stuff here.